src/index.js

```diff
@@ -1 +1,23 @@
-import './style.css';
\ No newline at end of file
+import './style.scss';
+import Leaderboard from './modules/leaderboard.js';
+
+const leaderboard = new Leaderboard();
+
+leaderboard.display();
+
+const refresh = document.querySelector('#refresh');
+const add = document.querySelector('#submit');
+
+refresh.addEventListener('click', () => {
+  leaderboard.display();
+});
+
+add.addEventListener('click', () => {
+  const name = document.querySelector('#name');
+  const score = document.querySelector('#score');
+  const scoreItem = {
+    name: name.value,
+    score: score.value,
+  };
```

Comment:
- [OPTIONAL] Please it will be nice to clear your inputs after submitting the scores, so that the same inputs are not submitted several times.

Reply:
Thanks for catching that! I've done it.